frontend/src/component/executiveDashboard/ExecutiveDashboard.tsx

```diff
@@ -45,7 +45,19 @@ const ChartWidget = styled(Widget)(({ theme }) => ({
     },
 }));
 
+const StickyWrapper = styled(Box)<{ scrolled?: boolean }>(
+    ({ theme, scrolled }) => ({
+        position: 'sticky',
+        top: 0,
+        zIndex: 1000,
+        padding: scrolled ? theme.spacing(2, 0) : theme.spacing(0, 0, 2),
```

Comment:
I'm curious what this does in practice? Why does the padding need to be different for the header once we scroll?

Reply:
When this gets pinned to to the top it looks weird with 0 padding (the search input label is almost half visible).  When you are at the top - even the 2 px padding makes it inconsistent with other pages with empty breadcrumbs.  This with the small transition makes it look ok. I added a video of how it looks